packages/design-system/src/scss/variables/colors.scss

```diff
@@ -5,6 +5,7 @@ $hover-background-color: #e1e9ec;
 $hover-basic-color: #f9fbfb;
 $hover-primary-color: #4379d6;
 $hover-red-color: #b9484a;
+$hover-font-primary-blue-color: #295bb2;
```

Comment:
@kamilmateusiak please review colours 🙏

Reply:
> ![image](https://user-images.githubusercontent.com/24878280/83001185-add93880-a00b-11ea-9b34-7a1668ce4de6.png)
> Need to ask - we are fine with this outline?

It was based on [Zeplin project](https://zpl.io/VYBq7xk):
![image](https://user-images.githubusercontent.com/29309941/83002766-b7fc3680-a00d-11ea-9454-148f0ba85023.png)

BTW Justyna will do the design review after demo :-)

Reply:
there is `$hover-primary-color: #4379d6;` already, no need for a new and pretty similar color

Reply:
But it's not the same as in design... I'm not sure what is the drill here, should we keep fewer colors to easier unify them later or stick with design?

Reply:
yeah, the color is slightly different so the chance there is an issue with design is high. Please ask jo about that. Maybe those colors could be merged

Reply:
kind reminder: ask jo about this color :)

Reply:
bb8c3a99f12b1b5c63a52627f4570cc29add5bca